packages/create-cloudflare/templates-experimental/angular/templates/wrangler.toml

```diff
@@ -1,5 +1,5 @@
 #:schema node_modules/wrangler/config-schema.json
 name = "<TBD>"
 compatibility_date = "<TBD>"
-main = "./dist/cloudflare/_worker.js"
-assets = { directory = "./dist/cloudflare", binding = "ASSETS" }
+main = "./dist/server/server.mjs"
+experimental_assets = { directory = "./dist/browser", binding = "ASSETS" }
```

Comment:
`experimental_assets` is renamed to `assets` today

Reply:
Doh!

Reply:
GOOD CATCH!